packages/wasm-dpp/src/data_contract_factory/data_contract_factory.rs

```diff
@@ -32,7 +32,7 @@ impl DataContractFactoryWasm {
         protocol_version: u32,
         external_entropy_generator_arg: Option<ExternalEntropyGenerator>,
     ) -> DataContractFactoryWasm {
-        if let Some(external_entropy_generator) = external_entropy_generator_arg {
```

Comment:
This condition doesn't make any sense

Reply:
🤷

Reply:
Just remove it

Reply:
73e1eb3664b652bd431ccb0bf56370ee81b7380e